webapp/channels/src/components/admin_console/workspace-optimization/dashboard_checks/config.ts

```diff
@@ -74,6 +76,40 @@ const sessionLength = (
     };
 };
 
+const fileStorage = async (
+    config: Partial<AdminConfig>,
+    formatMessage: ReturnType<typeof useIntl>['formatMessage'],
+    options: Options,
+) => {
+    const testFileStorage = async () => {
+        const pingResponse = await Client4.ping(true);
```

Comment:
This is definitely not specific to this PR, but it feels really weird that the ping API is used for these sort of checks. I know we've added more information to it over time, but I think it was originally just to check if the server was alive so mobile/desktop would know it was a valid server, and I wouldn't expect it to be used for stuff like this

Reply:
I would like to see the endpoint getting split into two:
1. One login-related one that also contains information like  `CanReceiveNotifications` or Android and iOS-supported versions.
2. A health check one, that is used for probing in Cloud and the dashboard.

Reply:
Yeah, that's what I'd expect too. That's obviously not something we can change now though